### PR TITLE
(MyBMW] command "horn-blow" not "horn-low"

### PR DESCRIPTION
**Pull request by igitti**
The command is "horn-blow" not "horn-low".
Tested with my car: "horn-blow" works, "horn-low" does not.

### CODE REVIEW

**Comment by lolodomo** [comment]
Looking very quickly at the code, I think there is another mismatch for another command: this is not "flash-lights" but "light-flash". Can you please fix that in README, it is only 4 lines above your current change.

File: bundles/org.openhab.binding.mybmw/README.md [hunk 1]
```diff
@@ -356,7 +356,7 @@ The channel _command_ provides options
 - _vehicle-finder_
 - _door-lock_
 - _door-unlock_
-- _horn-low_
+- _horn-blow_
```
Comment (lolodomo) [inline bundles/org.openhab.binding.mybmw/README.md:359]: Looking very quickly at the code, I think there is another mismatch for another command: this is not "flash-lights" but "light-flash". Can you please fix that in README, it is only 4 lines above your current change.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.mybmw/README.md (hunk 1 above)
Comment (igitti) [reply bundles/org.openhab.binding.mybmw/README.md:359]: Right, it must be "light-flash" not "flash-lights".
I fixed the README.md

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**